### PR TITLE
Enhancement/move ability dialog

### LINKED ISSUE #80: Choose Ability Modifiers appears on all story dialogues [MediumPriority]
**Bug Summary**
Whenever a story dialogue appears, the adjust ability modifiers page appears after

**Test Case(s)**
Travel up to floor 3 with the shop on it and it appears

---

**Expected behavior**
It should only appear once at the start of the game

**Observed Behaviour**
It happens after each story dialogue

**Steps To Reproduce**
1. Start the game
2. Make your way to floor 3 with the shop
3. Upon reaching that floor, the story dialogue will appear, followed by th option to change your ability modifier

**Environment**
- Version: Latest commit 4833bb8 
- OS: Windos
- Browser: Microsoft Edge

---

**Screenshots**

bug5
bug6

### PR DESCRIPTION
**Pull request by Pyxxil**
GitHub Issue (If applicable): #80 

## PR Type

What kind of change does this PR introduce?
- Bugfix
- Refactoring (no functional changes, no api changes)

## What is the current behavior?
Currently, when the user enters the third floor, the ability dialog will appear. The ability dialog also appears after the game text.

## What is the new behavior?
The ability dialog has been moved to be displayed to the user before the instructions have been displayed to them. This also fixes the dialog being displayed in later levels unintentionally.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current upstream repo
- [ ] Added new tests to the test suite for the feature
- [ ] Tested the full test suite, and ensured that the feature does not break current build.
- [ ] Docs have been added/updated which fit (for bug fixes / features)
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal)
- [ ] Checked licensing of Frameworks/Libraries (if applicable) 

Internal Issue (If applicable): closes #80

### CODE REVIEW

File: src/features/dialog-manager/dialogs/ability-dialog/index.js [hunk 1]
```diff
@@ -17,15 +17,14 @@ import incrementDexterity from './actions/increment-dexterity';
 import decrementCharisma from './actions/decrement-charisma';
 import incrementCharisma from './actions/increment-charisma';
 
-import setAbilityScores from '../../actions/set-ability-scores';
-import closeDialog from '../../actions/close-dialog';
+import confirmAbilityScoreDialog from '../../actions/confirm-ability-score-dialog';
```
Comment (Matteas-Eden) [inline src/features/dialog-manager/dialogs/ability-dialog/index.js:20]: The import here doesn't match the export from the file. Seems strange to have a mismatch.

**Review by Matteas-Eden — CHANGES_REQUESTED** [review]
Mostly LGTM, but just one small comment.

File: src/features/dialog-manager/dialogs/ability-dialog/index.js (hunk 1 above)
Comment (Pyxxil) [reply src/features/dialog-manager/dialogs/ability-dialog/index.js:20]: Technically it doesn't matter from what I understand, but I agree so I've changed it anyways.

**Review by Matteas-Eden — APPROVED** [review]
Tested locally and LGTM, thanks for addressing my feedback.

**Pull request merged**